packages/ipfs-core/src/index.js

```diff
@@ -108,11 +60,19 @@ async function create (options) {
   const startedApi = options.start && initializedApi && await initializedApi.start()
 
   /**
-   * @template T, THEN, ELSE
-   * @typedef {NonNullable<T> extends false
-   * ? THEN : ELSE } IsFalse
+   * create returns object that has different API set based on `options.init`
```

Comment:
We've talked about this before - the API returned should be the same.  Would it be possible to fix this bug now that TS will point out what's different?

Reply:
I would like to fix it, but as a followup change because this is large enough in scope that I'm actively trying to cut all the corners I can.